recirq/quantum_chess/circuit_transformer_test.py

```diff
@@ -33,59 +33,64 @@
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
 
 
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_three_split_moves(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
     c = cirq.Circuit(qm.split_move(a1, a2, b1), qm.split_move(a2, a3, b3),
                      qm.split_move(b1, c1, c2))
-    transformer.qubit_mapping(c)
-    device.validate_circuit(transformer.transform(c))
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
 
 
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_disconnected(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
     c = cirq.Circuit(qm.split_move(a1, a2, a3), qm.split_move(a3, a4, d1),
                      qm.split_move(b1, b2, b3), qm.split_move(c1, c2, c3))
-    transformer.qubit_mapping(c)
-    device.validate_circuit(transformer.transform(c))
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+#    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+#    device.validate_circuit(t.transform(c))
 
 
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_move_around_square(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
     c = cirq.Circuit(qm.normal_move(a1, a2), qm.normal_move(a2, b2),
                      qm.normal_move(b2, b1), qm.normal_move(b1, a1))
-    transformer.qubit_mapping(c)
-    device.validate_circuit(transformer.transform(c))
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+#    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+#    device.validate_circuit(t.transform(c))
 
 
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_split_then_merge(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
     c = cirq.Circuit(qm.split_move(a1, a2, b1), qm.split_move(a2, a3, b3),
                      qm.split_move(b1, c1, c2), qm.normal_move(c1, d1),
                      qm.normal_move(a3, a4), qm.merge_move(a4, d1, a1))
-    transformer.qubit_mapping(c)
-    device.validate_circuit(transformer.transform(c))
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+#    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+#    device.validate_circuit(t.transform(c))
```

Comment:
Should we add tests for some of the public functions?  For instance, adding functionality to test least_connected_qubit should be simple enough.

Reply:
yah, tests are added in `initial_mapping_utils_test.py` (which was previously `dynamic_look_ahead_heuristic_circuit_transformer_test.py`).